app/src/test/java/com/anysoftkeyboard/keyboards/views/AnyKeyboardViewWithMiniKeyboardTest.java

```diff
@@ -629,7 +629,7 @@ public void testLongPressWithPopupDoesNotOutputPrimaryCode() throws Exception {
     public void testLongPressKeyWithoutAny() throws Exception {
         Assert.assertNull(mViewUnderTest.getMiniKeyboard());
         Assert.assertFalse(mViewUnderTest.mMiniKeyboardPopup.isShowing());
```

Comment:
I would try:
```
final Keyboard.Key keyWithoutPopups = findKey('v');
keyWithoutPopups.popupCharacters = null;
```

Reply:
Your suggestion isn't working. It raises this exception:

```
com.anysoftkeyboard.keyboards.views.AnyKeyboardViewWithMiniKeyboardTest > testLongPressKeyWithoutAny FAILED
    java.lang.AssertionError: expected:<0> but was:<2131886131>
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.failNotEquals(Assert.java:834)
        at org.junit.Assert.assertEquals(Assert.java:645)
        at org.junit.Assert.assertEquals(Assert.java:631)
        at com.anysoftkeyboard.keyboards.views.AnyKeyboardViewWithMiniKeyboardTest.testLongPressKeyWithoutAny(AnyKeyboardViewWithMiniKeyboardTest.java:636)
```

Commenting out line 636 results in line 642 throwing an AssertionError instead. Commenting that one out too makes the build work, but I don't think the test is supposed to work like that.